app/services/import/determine-supplementary-billing-flags.service.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+/**
+ * Determines if an imported licence should be flagged for supplementary billing
+ * @module DetermineSupplementaryBillingFlagsService
+ */
+
+const ProcessImportedLicenceService = require('../licences/supplementary/process-imported-licence.service.js')
+const LicenceModel = require('../../models/licence.model.js')
+
+/**
+ * Determines if an imported licence should be flagged for supplementary billing.
+ *
+ * This service is responsible for determining whether a licence imported from NALD should be flagged for
+ * supplementary billing. It compares the licences end dates (such as lapsed, revoked or expired dates) between WRLS
+ * and the transformed data, and if there is a change in the dates allows the licence to go on to determining the flags.
+ *
+ * @param {object} naldLicence - the legacy NALD licence
+ * @param {object} wrlsLicenceId - the WRLS licence data
+ */
+async function go (naldLicence, wrlsLicenceId) {
+  try {
+    const licenceChanged = await _licenceChanged(naldLicence, wrlsLicenceId)
+
+    if (!licenceChanged) {
+      return
+    }
+
+    await ProcessImportedLicenceService.go(naldLicence, wrlsLicenceId)
+  } catch (error) {
+    global.GlobalNotifier.omfg('Supplementary Billing Flag failed for licence ', wrlsLicenceId, error)
```

Comment:
Need to update this call.

First, the message can be changed to make it clearer where the error has occurred, for example, "Determine supplementary billing flags on import failed".

Second, we tend to stub the notifier in our tests and then just confirm they got called with the arguments we expected.

However, if you look at the notifier code you'll see this will cause an error. `omfg()` expects an object for the second arg. So, you'll need to call it as

```javascript
  global.GlobalNotifier.omfg('Supplementary Billing Flag failed for licence ', { licenceId }, error)
```